search.php

```diff
@@ -3,25 +3,39 @@
 use Muppets\Classes\Db;
 use Muppets\Classes\MuppetDisplay;
 use Muppets\Classes\MuppetHydrator;
+use Muppets\Classes\MuppetSearch;
 
 require_once 'vendor/autoload.php';
+$errorInput = '';
+$errorDb = '';
+$muppetDisplay = '';
 
-$searchQuery = $_GET['searchInput'];
-
-$dbConn = new Db();
-$db = $dbConn->getDb();
-$muppets = MuppetHydrator::retrieveSearchQuery($db, $searchQuery);
-$muppetDisplay = MuppetDisplay::displayMuppets($muppets);
-
-//$error = '';
-//if (isset($_GET['error']) && $_GET['error'] === '1') {
-//    $error = '404 Muppet not found - you\'s a muppet!';
-//}
-// VALIDATE USERINPUT
-if ($searchQuery !== '') {
-    $displaysearchQuery = '<h2 class="searchTermPlaceholder" >You Searched For: </h2><h2 class="searchResult" >' . $searchQuery . '</h2>';
-}  else {
-    $displaysearchQuery = '';
+if (isset($_GET['searchInput'])){
+    $searchInput = $_GET['searchInput'];
+} else {
+    header('Location: index.php');
+}
+
+if ($searchInput !== '') {
+    $displaySearchInput = '<h2 class="searchTermPlaceholder" >Search Term: </h2><h2 class="searchResult" >' . $searchInput . '</h2>';
```

Comment:
- We need $searchInput to be replaced with $sanitisedSearchInput 
- this block of code needs to be moved below where $sanitisedSearchInput is defined